app/facades/dashboard_facade.rb

```diff
@@ -0,0 +1,9 @@
+class DashboardFacade
+  attr_reader :user_info, :scores
+
+  def initialize(current_user)
+    @user_info = FlatService.get_user(current_user)
```

Comment:
We'll have a better idea of this later, wonder if we'll want to have individual score objects down the road, not sure yet though